### DISCUSSION TITLE
MSI windows pop up during installation with Managed Bootstrapper

### DISCUSSION THREAD

**Discussion by Danielku15** [wixtoolset/issues#7629] [Questions]
We lately have seen that (without change in the installer) we get individual windows of the MSIs popping up during the installation. I assume this could be due to certain WIndows Policies that our IT has applied but I would like to understand: 

1. Are there maybe settings impacting whether the individual MSI windows applied by the bootstrapper is shown or not? 
2. Are you aware of related Windows (Security) Policies which might prevent applications to hide Windows of launched applications? 

We also launching some dynamically generated batch files from the .net code (setting UseShellExecute=true and CreateNoWindow=true) but still the batch files is shown visible. I thought this might be the same root cause you are familiar with. 

Executing the same code in a normal .net executable doesn't give this behavior (the windows are hidden) that's why I ask here.

**Comment by Danielku15** [comment]
Any idea or feedback on this topic? We don't want these MSI windows to be visible in our installer.

**Comment by robmen** [comment]
We are missing key information to really be helpful. For example, are you using Burn or a custom executable to install your MSI?

**Comment by Danielku15** [comment]
@robmen We are using a custom managed bootstrapper via `<BootstrapperApplicationRef Id="ManagedBootstrapperApplicationHost.RtfLicense">`. We just had a very interesting discovery as part of another bug which might shed some light to the problem: 

1. The problem happens only if we code-sign our installation files. We have custom targets for `SignMsi`, `SignContainers`, `SignBundleEngine` and `SignBundle` where we sign the input file.
2. We also noticed that none of the MSIs produce the typical log files in `%temp%`. We can see in the main bootstrapper log file that it sets variables like `WixBundleLog_<Msi Name>` to correct paths, but those files are not 

I also just noticed that in the original targets [here](https://github.com/wixtoolset/wix3/blob/develop/tools/WixBuild.Signing.targets#L33-L39) MSIs are not signed. Could it be our mistake that we sign everything and this shouldn't be done?

**Reply by robmen** [reply]
This sounds all very mysterious and will need a deep dive into the log files to really track down what is going awry. What you've described thus far is too random, so I don't have any ideas off the top of my head.
